### PR TITLE
add instance normalization 

### PR DESCRIPTION
**Pull request by candlewill**
According to @ggsonic 's experiment result, instance normalization could lead to a better result.

This commit adds a `norm_type` parameter in hyperparams.py. If `norm_type='bn'`, then use batch normalization, if `norm_type='ins'`, then instance normalization is used.

### CODE REVIEW

**Comment by Kyubyong** [comment]
Great idea!
